src/OpenLoco/MessageManager.cpp

```diff
@@ -1,13 +1,16 @@
 #include "MessageManager.h"
+#include "Date.h"
 #include "GameState.h"
 #include "Interop/Interop.hpp"
 #include "OpenLoco.h"
+#include "Ui/WindowManager.h"
 
 using namespace OpenLoco::Interop;
 
 namespace OpenLoco::MessageManager
 {
     static auto& rawMessages() { return getGameState().messages; }
+    static auto& numMessages() { return getGameState().numMessages; }
 
     Message* get(MessageId id)
```

Comment:
maybe rename to `getMessage`?

Reply:
It is in the namespace MessageManager so you already know it's getting a message